pkg/kotsadmsnapshot/backup.go

```diff
@@ -612,6 +897,11 @@ func ListInstanceBackups(ctx context.Context, kotsadmNamespace string) ([]*types
 			continue
 		}
 
+		// TODO(improveddr): support for improved DR in UI
+		if GetInstanceBackupType(veleroBackup) == InstanceBackupTypeApp {
```

Comment:
wouldn't the improved DR velero backups always have the `kots.io/instance` annotation that is checked right above this? is this even needed?

Reply:
instance type application backups would still have the `kots.io/instance` annotation, no?

Reply:
removed